source/Hit:Stand.py

```diff
@@ -0,0 +1,17 @@
+def Hit_Stand(self, Player):
+  def_input = input("Would you like to Hit or Stand?  ")
+    while def_input not in ["Hit", "Stand"]:
```

Comment:
If you use lowercase it will not be accepted because the list on line 3 does not include lowercase options.
alternatively you could use def_input.upper() to make all input upper case so your list could just be [ HIT , STAND ].

Reply:
that's a good idea. Ill plan on doing that!